.travis.yml

```diff
@@ -37,7 +37,8 @@ jobs:
     - make install-tools
     before_script:
     - make etcd
-    - make server
+    - make north-server
+    - make south-server
```

Comment:
I don't think for CI (travis) we need both NB and SB deployments. One of them should be enough

Reply:
Thanks,Done